distribution/docs/src/main/resources/_contents/_developing/developing-catalog-api-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+=== Catalog Development Fundamentals
+
+This section introduces the fundamentals of working with the Catalog API the OGC Filter for Queries.
+
+==== Simple Catalog API Implementations
+
+The Catalog API implementations, which are denoted with the suffix of `Impl` on the Java file names, have multiple purposes and uses.
```

Comment:
Do we end sentences like this - followed by bulleted lists of explication - with periods or colons?